scriptmodules/supplementary/runcommand/runcommand.sh

```diff
@@ -79,7 +79,11 @@
 
 ROOTDIR="/opt/retropie"
 CONFIGDIR="$ROOTDIR/configs"
-LOG="/dev/shm/runcommand.log"
+OUTPUTDIR="/dev/shm"
+[ -d "${OUTPUTDIR}" ] \
+  && [ -w "${OUTPUTDIR}" ] \
+    || OUTPUTDIR=/tmp
```

Comment:
Note we use `[[` for RetroPie bash scripts.

Reply:
It's a bit of a moot comment since you're not merging it unless you want to be picky.